pkgs/development/python-modules/mpv-jsonipc/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests
+, tqdm, websocket_client, pytest, pillow, mock, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "mpv-jsonipc";
+  version = "unstable-2020-02-19";
+
+  src = fetchFromGitHub {
+    owner = "iwalton3";
+    repo = "python-mpv-jsonipc";
+    rev = "b9bb0aafc0a454f9b33d655808329c0c2f191259";
+    sha256 = "04h6k1h1km9nlhdg60p5c1xci4qpr3hdydrp3cxa3qp137iffc5k";
+  };
+
+  # doesn't include tests
```

Comment:
Could you please write a reason for disabling the tests?

Reply:
I've clarified the comment. The python package itself does not contain tests, I'm not disabling, just following the pattern I was encouraged to follow for other python PRs.